### ISSUE TITLE
Distinguish 16S-only records from those that were extracted from whole genomes

### ISSUE THREAD

**Issue by marykstewart** [nhoffman/ya16sdb#32]
Hi Noah and Chris, is the source (NCBI 16S-only vs NCBI whole genome extracted) easily mapped to the records in the database, such that it would be possible to control the records displayed using that criteria (or have them marked in some way)?

**Comment by crosenth** [comment]
I think you determine this from the record accession https://www.ncbi.nlm.nih.gov/Sequin/acc.html

**Comment by marykstewart** [comment]
Yep, totally true, I can do that outside of ya16sdb...I was hoping for a filter within the application that would permit visualizing only records derived from whole genomes in the interface.

**Comment by crosenth** [comment]
Related to https://github.com/nhoffman/ya16sdb/issues/36

**Comment by crosenth** [comment]
Related Issue https://github.com/nhoffman/ya16sdb/issues/36 marked as Completed
